src/Backend/Modules/Users/Resources/config/services.yml

```diff
@@ -0,0 +1,4 @@
+services:
+    users.cli.reset_password:
```

Comment:
you can write the class name as service name and then it will automatically use that as the class so you don't need that argument